src/Payments.Jobs.MoneyMovement/appsettings.json

```diff
@@ -15,7 +15,10 @@
     "ClearingChart": "3",
     "ClearingAccount": "XXCLEAR",
     "FeeChart": "3",
-    "FeeAccount": "XXXXFEE"
+    "FeeAccount": "XXXXFEE",
+    "ClearingFinancialSegmentString": "313110-13U20-ADNO003-410004-43-000-0000000000-000000-0000-000000-000000",
+    "FeeFinancialSegmentString": "313110-13U20-ADNO003-410004-43-000-0000000000-000000-0000-000000-000000",
```

Comment:
I assume these are safe to have in source control.

Reply:
Yeah, just a couple of the new COA string.